### ISSUE TITLE
Limited screen reader support for spinner component

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3832]
## Describe the bug

The spinner component aria-live regions aren't working in some screen readers, notably NVDA and JAWS on PC browsers. The problem seems to be the dynamic nature of the aria-live region, and the suggestion is to make it always present. However, I do not know how it will behave when used inside of components that create templates on the fly, so we need to validate that. Probably also applies to progress bar.

See for more https://github.com/vmware/clarity/issues/3565

## How to reproduce

https://clarity.design/documentation/spinner

1.  Try to read with PC and JAWS/NVDA

## Expected behavior

Aria live region should be read.

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: 2+

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

**Comment by gnomeontherun** [comment]
dup of #3832
